### PR TITLE
add icon for DingTalk

### PR DESCRIPTION
**Pull request by Autines**
Select the appropriate sub-template:
* [**Icon addition**](?expand=1&template=icon_addition.md&labels=needs+review,icon+change)
DingTalk（com.alibaba.android.rimet/com.alibaba.android.rimet.biz.LaunchHomeActivity）

### CODE REVIEW

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: Screenshot 2023-12-10 at 12 15 32
Add a space here to make it like other entries.

File: svgs/dingtalk.svg
Comment (x9136) [inline svgs/dingtalk.svg:1]: Screenshot 2023-12-10 at 12 10 02
The size is wrong. Instead of 172, it should be 160.

**Review by x9136 — CHANGES_REQUESTED** [review]
Hey! Thanks for contributing, some minor fixes are needed.

File: svgs/dingtalk.svg
Comment (Autines) [reply svgs/dingtalk.svg:1]: Sorry,my mistake. Please check again.

**Pull request merged**